package.json

```diff
@@ -4,6 +4,6 @@
     "cucumber": "^7.0.0-rc.0",
     "dotenv": "^16.0.1",
     "isomorphic-fetch": "^3.0.0",
-    "pg": "^8.8.0"
+    "pg": "^8.9.0"
```

Comment:
@wbaccinelli I am not sure why this is different, I do not recall changing anything here.

Reply:
Let's not push this then